Works now on HTML5.

Vascofr opened the pull request.

Vascofr committed:
Works now on html5.

Geokureli commented on UserInterface/FileBrowse/source/PlayState.hx:
```diff
@@ -17,6 +17,11 @@ import flash.display.Loader;
 import flash.display.LoaderInfo;
 import flash.net.FileReference;
 import flash.net.FileFilter;
+#elseif html5
```
this should just be `#if (flash || html5)` openfl and flash imports are interchangeable in openfl/flixel

see here:
https://github.com/openfl/openfl/blob/develop/src/openfl/net/FileReference.hx#L1433
and here:
https://github.com/openfl/openfl/blob/develop/lib/flash-externs/src/flash/net/FileReference.hx#L31

Vascofr replied:
Thanks!

Vascofr committed:
Update PlayState.hx

Importing from openfl for both flash and html5 targets, since openfl and flash imports are interchangeable.

Geokureli commented:
@Vascofr did you specifically choose master over dev, or was master the default choice? please make PRs to dev, in the  future

Vascofr commented:
@Geokureli I'm not sure, but I suppose it was the default since I don't see why I'd choose master. Sorry for the mistake, I'll try to remember next time.

Geokureli commented:
only asking to see if can do anything on my end to prevent it. thanks!
